build.gradle

```diff
@@ -149,21 +149,7 @@ task createCOMPSnapshots {
 
 task checkCOMPSnapshots {
 
-    doFirst {
```

Comment:
Is this removal intentional?

Reply:
Not intentional - was just for initial testing - added back